server/dirstore.go

```diff
@@ -135,16 +145,17 @@ func NewDirJWTStore(dirPath string, shard bool, create bool) (*DirJWTStore, erro
 // evictOnLimit determines the behavior once limit is reached.
 //     true - Evict based on lru strategy
 //     false - return an error
-func NewExpiringDirJWTStore(dirPath string, shard bool, create bool, expireCheck time.Duration, limit int64,
+func NewExpiringDirJWTStore(dirPath string, shard bool, create bool, delete deleteType, expireCheck time.Duration, limit int64,
```

Comment:
Since those are public (exported) functions, but not yet released, should we have an option object/array of functions here to be able to extend in the future?

Reply:
Yes I can add that. array or map[string]interface{}?

Reply:
went with  `_ ...interface{}`

Reply:
No, I meant to pass an "Options" structure that we can add new options, or functions like we do for say nats.Connect(), etc..